setup.cfg

```diff
@@ -0,0 +1,51 @@
+[metadata]
+name = dbt-duckdb
+author = Josh Wills
+author-email = joshwills+dbt@gmail.com
+url = https://github.com/jwills/dbt-duckdb
+summary = The duckdb adapter plugin for dbt (data build tool)
+description_file = README.md
+long_description_content_type = text/markdown
+license = Apache-2
+classifier =
+    Development Status :: 5 - Production/Stable
+    License :: OSI Approved :: Apache Software License
+    Operating System :: Microsoft :: Windows
+    Operating System :: MacOS :: MacOS X
+    Operating System :: POSIX :: Linux
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3.11
+keywords =
+    setup
+    distutils
+
+[options]
+install_requires=
+    dbt-common>=1,<2
+    dbt-adapters>=1,<2
+    duckdb>=1.0.0
+    # add dbt-core to ensure backwards compatibility of installation, this is not a functional dependency
+    dbt-core>=1.8.0
+    pbr>=6.0.0
```

Comment:
Wait is this right? The user of the package doesn't need to install `pbr`, right? It's just a dev dependency?

Reply:
Ah good point
I was using `pbr` to get the package version in `version.py` but I can just get that with `importlib` instead. will update